### PR TITLE
[GPII-3642]: Add :grant_org_admin, :revoke_org_admin tasks

### PR DESCRIPTION
**Pull request by natarajaya**
This PR changes `:grant_owner_role`, `:revoke_owner_role` tasks into tasks that along with owner role in current project grant / revoke org-level super permissions (which are required for SCC administration operations).

The downside of current implementation is that now there is no way to automatically revoke org-level permissions, since we do not enforce org-level IAM policy. I'll think what can be done to improve this.

### CODE REVIEW

**Comment by natarajaya** [comment]
Update: issue of automatically revoking org-level user permissions is being taken care of in #366.

**Comment by stepanstipl** [comment]
Could this be 2 separate tasks - keep current `grant_owner_role` as it is, as it's project specific, and then create a new task for the organization-level permissions? It would be more in the principle of least privilege spirit.

I think the use case is that you would want one or the other, but not both at the same time, or?

**Comment by natarajaya** [comment]
> Could this be 2 separate tasks - keep current grant_owner_role as it is, as it's project specific, and then create a new task for the organization-level permissions

@stepanstipl Makes sense – it was my original plan, not sure what made me change my mind :)

Also added guard rail to prevent removing owner role from yourself in your own dev project (as discussed today).

File: gcp/PERMISSIONS.md [hunk 1]
```diff
@@ -75,14 +75,20 @@ Accounts and permissions
...
 
-  * `rake grant_owner_role` Attaches the owner role to the current user account
-  * `rake revoke_owner_role` Removes the owner role from the current user account
+This can be done with the following rake commands:
 
-  In the case of a developer's dev project:
+* `rake grant_owner_role` attaches the owner role in the current project to the current user account.
+* `rake revoke_owner_role` removes the owner role in the current project from the current user account.
+* `rake grant_super_powers` attaches organization-level super user roles to the current user account.
+* `rake revoke_super_powers` removes organization-level super user roles from the current user account.
```
Comment (mrtyler) [inline gcp/PERMISSIONS.md]: I'm not a big fan of these names -- I doubt I will remember which of these affects which set of permissions.

How about `(grant|revoke)_admin` and `(grant|revoke)_org_admin`? Or `_project_admin` and `_org_admin`?

Comment (amatas) [reply gcp/PERMISSIONS.md]: +1 for choosing more specific names. Although I like the term "super_powers", it seems to me more explicit the term `*org_admin` and `*project_admin`.

Comment (stepanstipl) [reply gcp/PERMISSIONS.md]: +1 for better, more consistent names... although not sure if we shouldn't keep existing `*_ownre_role` as it's a) pretty descriptive and b) already there :)

**Comment by stepanstipl** [comment]
thanks for splitting this in two, LGTM.

> Also added guard rail to prevent removing owner role from yourself in your own dev project (as discussed today).

Nice!

File: gcp/PERMISSIONS.md (hunk 1 above)
Comment (natarajaya) [reply gcp/PERMISSIONS.md]: I also like existing `*_owner_role` task naming, but will follow Tyler's advice for consistency.

**Pull request merged**